2024-08-16 01:17:20,652 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-16 01:17:20,652 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-16 01:17:20,653 :INFO : pytest_pyfunc_call :Information statement
2024-08-16 01:17:20,653 :INFO : pytest_pyfunc_call :Information statement
2024-08-16 01:17:20,653 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-16 01:17:20,653 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-16 01:17:20,654 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-08-16 01:17:20,654 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-08-16 01:17:20,654 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-08-16 01:17:20,654 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-08-16 01:17:20,657 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-08-16 01:17:20,657 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-08-21 19:58:42,720 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-21 19:58:42,720 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-21 19:58:42,723 :INFO : pytest_pyfunc_call :Information statement
2024-08-21 19:58:42,723 :INFO : pytest_pyfunc_call :Information statement
2024-08-21 19:58:42,724 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-21 19:58:42,724 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-08-21 19:58:42,724 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-08-21 19:58:42,724 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-08-21 19:58:42,724 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-08-21 19:58:42,724 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-08-21 19:58:42,724 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-08-21 19:58:42,724 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-15 22:10:41,888 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 22:10:41,888 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 22:10:41,889 :INFO : pytest_pyfunc_call :Information statement
2024-09-15 22:10:41,889 :INFO : pytest_pyfunc_call :Information statement
2024-09-15 22:10:41,889 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 22:10:41,889 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 22:10:41,889 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-15 22:10:41,889 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-15 22:10:41,889 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-15 22:10:41,889 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-15 22:10:41,889 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-15 22:10:41,889 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-15 23:27:18,783 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 23:27:18,783 :INFO : pytest_pyfunc_call :Information statement
2024-09-15 23:27:18,783 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 23:27:18,783 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-15 23:27:18,783 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-15 23:27:18,783 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-15 23:27:43,373 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 23:27:43,373 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 23:27:43,374 :INFO : pytest_pyfunc_call :Information statement
2024-09-15 23:27:43,374 :INFO : pytest_pyfunc_call :Information statement
2024-09-15 23:27:43,374 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 23:27:43,374 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-15 23:27:43,375 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-15 23:27:43,375 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-15 23:27:43,375 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-15 23:27:43,375 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-15 23:27:43,375 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-15 23:27:43,375 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-16 00:34:25,635 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-16 00:34:25,635 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-16 00:34:25,635 :INFO : pytest_pyfunc_call :Information statement
2024-09-16 00:34:25,635 :INFO : pytest_pyfunc_call :Information statement
2024-09-16 00:34:25,636 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-16 00:34:25,636 :DEBUG : pytest_pyfunc_call :A debug statement is executed
2024-09-16 00:34:25,636 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-16 00:34:25,636 :WARNING : pytest_pyfunc_call :Something is in warning mode
2024-09-16 00:34:25,636 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-16 00:34:25,636 :ERROR : pytest_pyfunc_call :A Major error has happend
2024-09-16 00:34:25,636 :CRITICAL : pytest_pyfunc_call :Critical issue
2024-09-16 00:34:25,636 :CRITICAL : pytest_pyfunc_call :Critical issue
